tests/integration/index/create_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package index
+
+import (
+	"testing"
+
+	"github.com/sourcenetwork/defradb/request/graphql/schema"
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestIndexCreateWithCollection_ShouldNotHinderQuerying(t *testing.T) {
```

Comment:
suggestion: I think someone on the team (maybe John) hates underscores in test names.  I like them, and use them in how I would otherwise name my tests, but I think you should check in with the rest of the team RE this on discord/standup in case anyone with strong feelings misses it at review time.

Reply:
I'm fine with it as long as there is only one and the use is to separate the function being tested from extra details.

Reply:
Lol - why only one?  The test name format I am most familiar with uses two: `Action_Result_Input`, e.g. `Add_ReturnsFive_GivenThreeAndTwo`.  Used that all the way through my career prior to Source, and oddly enough, I think it was taken from a Google coding standards doc (I guess not one used by the Go team :)).

Reply:
My point is that I don't want to start seeing stuff like `Test_Index_Create_With_Collection...`.

`Action_Input_Result` could be fine too.